### ISSUE TITLE
Version 3.7.0 does not have slider bars for manual step size

### ISSUE THREAD

**Issue by BernieMurphy** [arkypita/LaserGRBL#1087]
I though I saw a screen shot with this feature.  Did I miss something?

Great program for beginner!

Thank you

**Comment by StuartB4** [comment]
I think you have Continuous jog enabled on the Jog Control tab in the Settings on the Grbl menu.

**Comment by BernieMurphy** [comment]
﻿Thank you for the prompt reply! 

Actually, I did have Continuous Jog off.  I turned it on and then off and then the slider bars appeared.  Looks like an uninitialized variable in the code?

Great program - works extremely well with CNC 3018 PRO with Woodpecker control board.

I will send a donation via PayPal.

Thank you!

Bernie Murphy, Ottawa, Canada


Sent from my iPad 7th generation
(most typos in this message have been introduced by highly sophisticated auto-correction software)
